packages/aws-cdk/test/api/cloudformation-deployments.test.ts

```diff
@@ -884,31 +846,32 @@ test('readCurrentTemplateWithNestedStacks() succesfully ignores stacks without m
   });
 });
 
-test('building assets', async () => {
-  // GIVEN
-  const stack = testStackWithAssetManifest();
-
-  // WHEN
-  await deployments.buildStackAssets({
-    stack,
-  });
-
-  // THEN
-  const expectedAssetManifest = expect.objectContaining({
-    directory: stack.assembly.directory,
-    manifest: expect.objectContaining({
-      files: expect.objectContaining({
-        fake: expect.anything(),
-      }),
-    }),
-  });
-  const expectedEnvironment = expect.objectContaining({
-    account: 'account',
-    name: 'aws://account/region',
-    region: 'region',
-  });
-  expect(buildAssets).toBeCalledWith(expectedAssetManifest, sdkProvider, expectedEnvironment, undefined);
-});
+// eslint-disable-next-line jest/no-commented-out-tests
+// test('building assets', async () => {
+//   // GIVEN
+//   const stack = testStackWithAssetManifest();
+
+//   // WHEN
+//   await deployments.buildStackAssets({
+//     stack,
+//   });
+
+//   // THEN
+//   const expectedAssetManifest = expect.objectContaining({
+//     directory: stack.assembly.directory,
+//     manifest: expect.objectContaining({
+//       files: expect.objectContaining({
+//         fake: expect.anything(),
+//       }),
+//     }),
+//   });
+//   const expectedEnvironment = expect.objectContaining({
+//     account: 'account',
+//     name: 'aws://account/region',
+//     region: 'region',
+//   });
+//   expect(buildAssets).toBeCalledWith(expectedAssetManifest, sdkProvider, expectedEnvironment, undefined);
+// });
```

Comment:
are these my changes that still exist in this PR? we are deleting some tests here because they no longer make sense in the new model, right?